src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -22,9 +23,14 @@ export const CollectorSignals: GraphQLFieldConfig<any, ResolverContext> = {
       },
     },
   }),
-  description: "List of collector signals",
+  description: "Collector signals on artwork",
 
-  resolve: async (artwork, {}, _ctx) => {
-    return artwork.collectorSignals
+  resolve: async (artwork, {}, ctx) => {
+    try {
+      const collectorSignals = await collectorSignalsLoader(artwork, ctx)
+      return collectorSignals
+    } catch (e) {
+      console.error("Error in CollectorSignals resolver: ", e)
```

Comment:
Is there a reason to `catch` and log, vs. not catching?

Reply:
When I didn't catch and just threw an error in the resolver I got an error instead of data in my query.

Reply:
![Screen Shot 2024-07-24 at 5 11 52 PM](https://github.com/user-attachments/assets/1b4b9323-588f-4633-acb9-84826b9a5e99)

You shouldn't `catch` - (per my comment yesterday).

Let the standard error catching happen. It will resolve `collectorSignals: null`, and everything will be fine. Don't `catch` unless you really _need_ to.

Reply:
Ah, you are totally right. I knew about `data`/`errors`, should have checked closer.